distribution_strategy/keras_model_to_estimator_client_alt.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""An example of training Keras model with multi-worker strategies."""
+from __future__ import absolute_import
```

Comment:
Is there a way for us to use keras_model_to_estimator_client.py with some flags instead of creating a new copy of the file?

Reply:
Almost half of `keras_model_to_estimator_client.py` is kubernetes code. From my perspective it's a good idea to have clean, minimalistic example that doesn't contain this code and focused on the question "what we actually need to run training in standalone mode".